.NET/Microsoft.Recognizers.Text.DateTime/Arabic/Parsers/ArabicDateParserConfiguration.cs

```diff
@@ -136,6 +136,12 @@ public ArabicDateParserConfiguration(ICommonDateTimeParserConfiguration config)
 
         public Regex BeforeAfterRegex { get; }
 
+        public Regex TasksModeNextWeekRegex { get; }
```

Comment:
As mentioned in the resources, this should be just one regex.

Reply:
resolved, created only one regex and pushed changes.